### ISSUE TITLE
tlg0557.tlg003.perseus-eng 

### ISSUE THREAD

**Issue by lcerrato** [PerseusDL/catalog_data#197]
I cannot edit the existing records.

So I want perseus-eng1 to be perseus-eng3 and eng2 to be eng4, but it looks like there was some bumping already done.

When I go here:
https://catalog.perseus.org/cite-collections/versions/search?utf8=✓&field_type=version&search=tlg0557.tlg003.perseus-eng

and try to edit perseus-eng4 to be Higginson, the data I see is actually for a Thucydides work (tlg0003.tlg001.opp-grc64). Very strange.

<img width="341" alt="image" src="https://github.com/user-attachments/assets/7ceb3656-dfae-41fd-ae65-2593d84123a8">

**Comment by AlisonBabeu** [comment]
hi @lcerrato so I have no idea what is going on here. Both of those Thucydides versions were invalidated and redirected during the grand URN purge of 2014-2015 where I was picking URNs for editions of multi-volume works where each MODS record for a volume had gotten its own URN. I also looked up both the top level CITEURNs: urn:cite:perseus:catver.82.1 and urn:cite:perseus:catver.83.1 across the catalog data and did not find them elsewhere.  To the best of my knowledge I also never bumped these URNS because I wouldn't do that without you having created new updated versions.

**Comment by AlisonBabeu** [comment]
So I just dug in deeper and this seems to be a larger problem with the URN reservation process in the CITE Collection tables in that for some reason it seems to be reusing CITEURNs. If you go to page 78 or the version table [here](https://catalog.perseus.org/cite-collections/versions?page=78) the last CITE URN is urn:cite:perseus:catver.15597.1 for a bumped version of Plutarch. If you click on"Show" you get the correct edition metadata.

Then for some reason you can't view page 79, but if you view page [80,](https://catalog.perseus.org/cite-collections/versions?page=80) the first new URN is urn:cite:perseus:catver.67.1, which is purportedly for urn:cts:greekLit:tlg0527.tlg040.perseus-eng2, a bumped version of Obadiah.  If you click on "Show" however, it takes you to the metadata for the original CITEURN, another Thucydides edition. It looks all the URNs on this page have invalid metadata when you click on "Show". They have the correct version CTS-URN but have maintained the original metadata for the CITE URN. 

I also didn't scroll down enough when I first searched on the [CITE URN: urn:cite:perseus:catver.83.1.](https://catalog.perseus.org/cite-collections/versions/search?utf8=%E2%9C%93&field_type=urn&search=urn%3Acite%3Aperseus%3Acatver.83.1) it shows two records, one for the original Thucydides and one for the work of Epictetus .

**Comment by lcerrato** [comment]
So aside from this, I don't think that these should be tlg0557.tlg003.perseus-eng files, because they incorporate fragments from tlg003, tlg004, and tlg005 (as well as elsewhere). I tried to pull out some that match tlg005, but there is no 1 to 1 match for the other files and it wasn't a good use of time.
Perhaps there should be a new ID for these translations? Not sure what to do.

**Comment by lcerrato** [comment]
Yes, it's really more than 003 and any alignment will simply break. The other option would be unused digits, so tlg0557.tlg008 or something?

**Comment by lcerrato** [comment]
There are 2 translations so not only do they have no relation to the Greek, they have limited overlap with one another. I'll do the 003a I think.
